core/src/components/col/col.tsx

```diff
@@ -55,73 +53,41 @@ export class Col implements ComponentInterface {
    */
   @Prop() offsetXl?: string;
 
-  /**
-   * The amount to pull the column, in terms of how many columns it should shift to the start of
-   * the total available.
-   */
-  @Prop() pull?: string;
-
-  /**
-   * The amount to pull the column for xs screens, in terms of how many columns it should shift
-   * to the start of the total available.
-   */
-  @Prop() pullXs?: string;
-  /**
-   * The amount to pull the column for sm screens, in terms of how many columns it should shift
-   * to the start of the total available.
-   */
-  @Prop() pullSm?: string;
-  /**
-   * The amount to pull the column for md screens, in terms of how many columns it should shift
-   * to the start of the total available.
-   */
-  @Prop() pullMd?: string;
-  /**
-   * The amount to pull the column for lg screens, in terms of how many columns it should shift
-   * to the start of the total available.
-   */
-  @Prop() pullLg?: string;
-  /**
-   * The amount to pull the column for xl screens, in terms of how many columns it should shift
-   * to the start of the total available.
-   */
-  @Prop() pullXl?: string;
-
-  /**
-   * The amount to push the column, in terms of how many columns it should shift to the end
-   * of the total available.
+/**
+   * The order of the column, in terms of where the column should position itself in the columns renderer.
+   * If no value is passed, the column order implicit value will be the order in the html structure.
    */
-  @Prop() push?: string;
```

Comment:
Can we deprecate the `push` properties instead of removing them?

Reply:
Let me circle that back and reply to you.

Reply:
As discussed, the properties were re-added, and deprecated. Additionally an warning was added when one of those properties are used:
<img width="1313" height="320" alt="warning-message" src="https://github.com/user-attachments/assets/0f2f373d-d72f-486d-bfea-dd1dbe2c7313" />